core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -25,14 +25,164 @@ var users = require('../protractor_utils/users.js');
+    enableMarkCorrectness();
+
+    // Go back to mark the solution as correct.
+    explorationEditorPage.navigateToMainTab();
+    explorationEditorMainTab.moveToState('First');
+    responseEditor = explorationEditorMainTab.getResponseEditor(0);
+    responseEditor.markAsCorrect();
+    explorationEditorMainTab.expectTickMarkIsDisplayed();
+    explorationEditorPage.saveChanges();
+    workflow.publishExploration();
+    testEnableCorrectnessInPlayerPage();
+  });
+
+  it('should allow selecting correct feedback from the response editor ' +
+     'during set the interaction', function() {
+    // Turn on correctness feedback first.
+    enableMarkCorrectness();
+
+    // Go to main tab to create interactions.
+    explorationEditorPage.navigateToMainTab();
+    explorationEditorMainTab.setStateName('First');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'Select the right option.'));
+
+    // Create interaction without closing the add response modal. Set
+    // correctness in the modal.
+    explorationEditorMainTab.setInteractionWithoutCloseAddResponse('TextInput');
+    responseEditor = explorationEditorMainTab.getResponseEditor('pop');
+    responseEditor.markAsCorrect();
+
+    // Set the response for this interaction and close it.
+    explorationEditorMainTab.setResponse(
+      'TextInput', forms.toRichText('Correct!'),
+      'End', true, 'Equals', 'One');
+
+    explorationEditorMainTab.expectTickMarkIsDisplayed();
+    responseEditor = explorationEditorMainTab.getResponseEditor('default');
+    responseEditor.setFeedback(forms.toRichText('Wrong!'));
+    explorationEditorMainTab.moveToState('End');
+    explorationEditorMainTab.setInteraction('EndExploration');
+    explorationEditorPage.saveChanges();
+    workflow.publishExploration();
+    testEnableCorrectnessInPlayerPage();
+  });
+
+  it('should allow selecting correct feedback from the default response editor',
+    function() {
+      // Turn on correctness feedback first.
+      enableMarkCorrectness();
+
+      // Go back to main tab to create interactions.
+      explorationEditorPage.navigateToMainTab();
+      explorationEditorMainTab.setStateName('First');
+      explorationEditorMainTab.setContent(forms.toRichText(
+        'Select the right option.'));
+      explorationEditorMainTab.setInteraction('NumericInput');
+
+      // Set correctness in response editor.
+      responseEditor = explorationEditorMainTab.getResponseEditor('default');
+      responseEditor.markAsCorrect();
+      responseEditor.setFeedback(forms.toRichText('Correct!'));
+      responseEditor.setDestination('End', true, true);
+      explorationEditorMainTab.expectTickMarkIsDisplayed();
+
+      explorationEditorMainTab.moveToState('End');
+      explorationEditorMainTab.setInteraction('EndExploration');
+      explorationEditorPage.saveChanges();
+      workflow.publishExploration();
+      testEnableCorrectnessInPlayerPage();
+    });
+
+  afterEach(function() {
```

Comment:
After each blocks should also assert that there are no console errors. Could you take a look at some of the existing test suites which do this check and replicate the same here?

Reply:
[Adding refrence]  In afterEach block assert that there are no console errors as suggested here: https://github.com/oppia/oppia/wiki/End-to-End-Tests#important-gotchas

Reply:
Did not notice that. Updated, thanks!